File: backend/src/main/java/com/svelteup/app/backend/payment/dtos/PaymentMethodDeleteDto.java
package com.svelteup.app.backend.payment.dtos;

import com.svelteup.app.backend.payment.models.AcceptedPaymentsEnum;
import io.swagger.annotations.ApiModel;

@ApiModel(value="JSON representing a token which customer uses to delete a payment method.")
public class PaymentMethodDeleteDto {
    public String paymentMethodToDelete;
}

File: frontend/src/Dto/payment/PaymentMethodDeleteDto.ts
import BaseDto from "../BaseDto";
import type ToApiDto from "../ToApiDto";

class PaymentMethodDeleteDto implements ToApiDto
{
    public paymentMethodToDelete:string;
    constructor(paymentMethodToDelete:string)
    {
        this.paymentMethodToDelete = paymentMethodToDelete;}

    toApiDto(): object {
        return this;
    }
}

export default PaymentMethodDeleteDto;